save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -91,7 +92,11 @@ abstract class Plugin(
             val foundTests = rawTestFiles.filter { rawTestFile ->
                 testFiles.any {
                     if (fs.exists(it.toPath())) {
-                        it in rawTestFile.test.toString()
+                        (it.toPath().parents().toList() + listOf(it.toPath())).all { testFile ->
```

Comment:
```suggestion
                        (it.toPath().parents().toList() + listOf(it.toPath())).all { part ->
```
This argument name makes no sense, it implies that you are traversing `testFiles`, but you are not